changelogs/fragments/5282-locale_gen.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - "locale_gen - fix support for Ubuntu (https://github.com/ansible-collections/community.general/issues/5281, https://github.com/ansible-collections/community.general/pull/5282)."
```

When there is an issue the PR is solving, we usually refer only to the issue, not the PR:
```suggestion
  - "locale_gen - fix support for Ubuntu (https://github.com/ansible-collections/community.general/issues/5281)."
```
This is not a hard requirement. Feel free to ignore if you prefer it like that.

---

Done

---

I usually prefer to mention both issue and PR, but that's individual preference.